backend/svirt.pm

```diff
@@ -174,6 +175,30 @@ sub load_snapshot ($self, $args) {
     return $post_load_snapshot_command;
 }
 
+sub do_extract_assets ($self, $args) {
+    my $first_hdd = $bmwqemu::vars{S390_ZKVM} ? 'a' : 'b';
+    my $name = $args->{name};
+    die "No name given" unless $args->{name};
+    $name =~ /\.([[:alnum:]]+)$/;
+    die "No HDD specified" unless $args->{hdd_num};
+    my $hdd_num = $args->{hdd_num} - 1;
+    die "No format specified" unless $args->{format};
+    my $format = $args->{format};
+    return undef if (($format ne 'raw') and ($format ne 'qcow2'));
+    my $svirt = $self->select_console('svirt');
+    my $svirt_name = $svirt->name . chr(ord($first_hdd) + $hdd_num);
+
+    my $image_storage = '/var/lib/libvirt/images';
+    my $svirt_img_name = $image_storage . '/' . $args->{svirt_name} . '.img';
```

Comment:
I think `$args->{svirt_name}` is not set. `$args` comes from `OpenQA::Isotovideo::Utils::handle_generated_assets`